packages/wonder-blocks-toolbar/components/toolbar.js

```diff
@@ -107,7 +114,7 @@ export default class Toolbar extends React.Component<Props> {
                                 sharedStyles.center,
                             ]}
                         >
-                            <TitleComponent id="wb-toolbar-title">
+                            <TitleComponent id={titleId || "wb-toolbar-title"}>
```

Comment:
IDs have to be unique so we certainly shouldn't hard code a fixed string as that would not by WCAG compliant.